Libraries/Components/ScrollView/RecyclerViewBackedScrollView.android.js

```diff
@@ -113,6 +113,18 @@ var RecyclerViewBackedScrollView = React.createClass({
       );
     });
 
+    const refreshControl = this.props.refreshControl;
+    if (refreshControl) {
+      // Wrap the NativeAndroidRecyclerView with a AndroidSwipeRefreshLayout.
+      return React.cloneElement(
+        refreshControl,
+        {style: props.style},
+        <NativeAndroidRecyclerView {...props}>
```

Comment:
I'm mostly concerned about passing props.style to both AndroidSwipeRefreshLayout and NativeAndroidRecyclerView. Styles like margin could cause issues because they will get applied twice. Adding style after ...props will prevent that.